### ISSUE TITLE
For loop within addCons?

### ISSUE THREAD

**Issue by martinerr** [SCIP-Interfaces/PySCIPOpt#183]
Hi,

I was wondering how it is possible to include a product of multiple decision variables in a constraint. 
So instead of the quicksum function I'm looking for a similar function to multiply some decision variables. Since I don't know if this even exists, I was wondering how you could include a for loop within a constraint.

This is what i have tried:
```
    for i in C:
        model.addCons(
        y[i]=1        
        for j in BD:
            y[i]*=x[j])
```

However it says invalid syntax.

I would really appreciate some help with this. Thanks!

**Comment by fserra** [comment]
Hi 

There is `quickprod` that works like quicksum. have a look at test/test_quickprod

**Comment by ramabile** [comment]
**TL;DR**: look at `test_quickprod.py`

Then...
It says invalid syntax because you can't write statements (as your [assignment `=` and augmented assignment `*=`](https://docs.python.org/3/reference/simple_stmts.html) or your [`for` loop](https://docs.python.org/3.6/reference/compound_stmts.html)) in function calls.

Without `quickprod` I would write this:
```# init
from pyscipopt import Model
model = Model()

# add variables
v = [model.addVar("v%s" % i) for i in range(10)]

# make product of variables
vprod = 1
for iv in v:
    vprod *= iv

# constraint product to 1
model.addCons(vprod == 1)
```

**Comment by mattmilten** [comment]
I suppose this can be closed now.
